packages/odyssey-react-mui/src/Field.tsx

```diff
@@ -31,7 +33,7 @@ export type FieldProps = {
   /**
    * The field type determines how ARIA components are setup. It's important to use this to denote if you expect only one component (like a text field) or multiple (like a radio group).
    */
-  fieldType: "single" | "group";
+  fieldType: (typeof fieldTypeValues)[number];
```

Comment:
Cool! It works like it did before with a union type. Nice!

![image](https://github.com/okta/odyssey/assets/97464104/f1b23128-a3fe-49a7-927a-1c7f83edf806)